databases/week1/homework.sql

```diff
@@ -0,0 +1,87 @@
+-- Find out how many tasks are in the task table
+select
+    COUNT(id)
+FROM
+    task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT
+    COUNT(id)
+from
+    task
+WHERE
+    due_date is Null;
+
+-- Find all the tasks that are marked as done
+SELECT
+    *
+FROM
+    task
+WHERE
+    status_id = "Done";
```

Comment:
Unfortunately this will not work as the `status_id` is a number and here it is compared against a string (text).

To solve this you should make a join statement with the `status` table based on the `task.status_id` and then use the `status.name = "Done"`